docs/UserGuide.md

```diff
@@ -343,7 +371,7 @@ Example:
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the employee list and task list.
+Delete all entries from the employee list and task list.
```

Comment:
Hi @alfredkohhh, you can change to "Deletes" for consistency :smiley:

Reply:
Thanks @dannytayjy ! I will change this to your suggestion 🍡